projects/core/src/auth/user-auth/services/auth-multisite-isolation.service.ts

```diff
@@ -49,4 +54,17 @@ export class AuthMultisiteIsolationService {
       map((decorator) => userId + decorator)
     );
   }
+
+  async isDifferentBaseSite(): Promise<boolean> {
+    const auth = this.statePersistenceService.readStateFromStorage({
+      key: 'auth',
+    }) as SyncedAuthState;
+
+    let baseSite = await this.getBaseSiteDecorator().toPromise();
+    baseSite = baseSite.replace(this.MULTISITE_SEPARATOR, '');
+
+    const tokenBaseSite = auth.redirectUrl?.split('/')[0];
+
+    return baseSite && baseSite !== tokenBaseSite ? true : false;
```

Comment:
I think `return baseSite && baseSite !== tokenBaseSite` is enough here.

Reply:
With this I am having an error, as we should return a `boolean`, but with this solution, a string seems to be possible. However, I simplified this line.